odoo/addons/base/models/res_currency.py

```diff
@@ -84,7 +84,8 @@ def _compute_current_rate(self):
             currency.rate = (currency_rates.get(currency.id) or 1.0) / last_rate[company]
             currency.inverse_rate = 1 / currency.rate
             if currency != company.currency_id:
-                currency.rate_string = '1 %s = %s %s' % (company.currency_id.name, currency.rate, currency.name)
+                currency.rate_string = ('1 %s = %.6' + ('f' if currency.rate > 1 else 'g') + ' %s') \
```

Comment:
Why would you want something else than fixed point in currency rates? When would you want scientific notation? It doesn't seem consistent with the rest of Odoo; and it is harder to read too.
https://docs.python.org/3/library/string.html#format-specification-mini-language

Reply:
Was a way to show 2 significant digits instead of 2 digits after 0 (0.0012 vs 0.00). Less needed with 6 digits.